pages/index.tsx

```diff
@@ -136,7 +136,7 @@ gtag('js', new Date());gtag('config', 'G-186QH0S5W3');`
 };
 
 
-export const getStaticProps: GetStaticProps<HomeProps> = async () => {
+export const getServerSideProps: GetServerSideProps<HomeProps> = async () => {
```

Comment:
Hey, we use Static props so that the page is background-refreshed, therefore the user doesn't need to wait for the page to load